internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4388,6 +4551,561 @@ func TestHandleVerifyChain(t *testing.T) {
+		"3ec0d959758d786d00"
+
+	var block wire.MsgBlock
+	blockD, err := hex.DecodeString(blockStr)
+	if err != nil {
+		t.Fatalf("unexpected decode err: %v", err)
+	}
+	err = block.FromBytes(blockD)
+	if err != nil {
+		t.Fatalf("unexpected block deserialization err: %v", err)
+	}
+
+	workStr := "070000000380b2db1300aabd5b1fb25821b0a3acb6773c5005b9b504f" +
+		"a076cff0b9000008a91e65f0d8f155dfc258f12540a08124ee3a69a8bc67cc98" +
+		"90b49ceb6bda2ce766e5181aa387977806d8bdb4ecea26d24a28dc7d0eedc230" +
+		"a689ea6442d51c1010000000000000000000000000000000d423c20204e00000" +
+		"00000004a00000085020000e4f9225f000000000000000000000000000000000" +
+		"0000000000000000000000000000000000000000000000080000001000000000" +
+		"00005a0"
+
+	headerD, err := hex.DecodeString(workStr)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	var work wire.BlockHeader
+	err = work.FromBytes(headerD)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	submission := "70000000380b2db1300aabd5b1fb25821b0a3acb6773c5005b9b5" +
+		"04fa076cff0b9000008a91e65f0d8f155dfc258f12540a08124ee3a69a8bc67c" +
+		"c9890b49ceb6bda2ce766e5181aa387977806d8bdb4ecea26d24a28dc7d0eedc" +
+		"230a689ea6442d51c1010000000000000000000000000000000d423c20204e00" +
+		"00000000004a00000085020000e4f9225fff190500ee00b5c100000000000000" +
+		"0000000000000000000000000000000000000000000000000080000001000000" +
+		"00000005a0"
+
+	invalidPOWSub := "70000000380b2db1300aabd5b1fb25821b0a3acb6773c5005b9" +
+		"b504fa076cff0b9000008a91e65f0d8f155dfc258f12540a08124ee3a69a8bc6" +
+		"7cc9890b49ceb6bda2ce766e5181aa387977806d8bdb4ecea26d24a28dc7d0ee" +
+		"dc230a689ea6442d51c1010000000000000000000000000000000d423c20204e" +
+		"0000000000004a00000085020000e4f9225fffffffffffffb5c1000000000000" +
+		"0000000000000000000000000000000000000000000000000000800000010000" +
+		"0000000005a0"
+
+	lessThanGetWorkDataLen := submission[10:]
+
+	invalidHexSub := "70000000380b2dg1300aabd5b1fb25821b0a3acb6773c5005b9b5" +
+		"04fa076cff0b9000008a91e65f0d8f155dfc258f12540a08124ee3a69a8bc67c" +
+		"c9890b49ceb6bda2ce766e5181aa387977806d8bdb4ecea26d24a28dc7d0eedc" +
+		"230a689ea6442d51c1010000000000000000000000000000000d423c20204e00" +
+		"00000000004a00000085020000e4f9225fff190500ee00b5c100000000000000" +
+		"0000000000000000000000000000000000000000000000000080000001000000" +
+		"00000005a0"
+
+	miningaddr, err := dcrutil.DecodeAddress(
+		"DsRM6qwzT3r85evKvDBJBviTgYcaLKL4ipD", defaultChainParams)
+	if err != nil {
+		t.Fatalf("[DecodeAddress] unexpected error: %v", err)
+	}
+
+	testRPCServerHandler(t, []rpcTest{
+		{
+			name:    "handleGetWork: CPU IsMining enabled",
+			handler: handleGetWork,
+			cmd:     &types.GetWorkCmd{},
+			mockCPUMiner: func() *testCPUMiner {
+				cpu := defaultMockCPUMiner()
+				cpu.isMining = true
+				return cpu
+			}(),
+			mockBlockTemplater: func() *testBlockTemplater {
+				return defaultMockBlockTemplater()
```

Comment:
I believe `defaultMockConfig` be setting the default to ensure it's always set and then there would be no need to override it here unless you specifically are changing something from the default.